examples/ruby/start

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+source ~/.rvm/scripts/rvm
+RUBY_VERSION="2.7.2"
+
+function announce() {
+  echo "++++++++++"
+  echo "$@"
+  echo "++++++++++"
+}
+
+if [[ ! -d "./out/ruby" ]]; then
+  announce "Client not found. Generating..."
+  ./bin/generate_client ruby > /dev/null
+  echo
+fi
+
+if [[ -z "$(docker-compose ps -q)" ]]; then
+  announce "Environment not found. Spinning up..."
+  ./bin/start_conjur > /dev/null
+  echo
+fi
+
+if [[ ! -f ./out/ruby/openapi_client-1.0.0.gem ]]; then
+  announce "Building OpenAPI gem..."
+  gem build ./out/ruby/openapi_client.gemspec -C ./out/ruby/
```

Comment:
We can actually move this logic into the dockerfile, which should make this whole script only dependent on docker!